picard/ui/itemviews/basetreeview.py

```diff
@@ -567,3 +568,82 @@ def moveCursor(self, action, modifiers):
     @property
     def default_drop_target(self):
         return None
+
+    def setup_filter_box(self):
+        self.filter_box = Filter(self)
```

Comment:
```suggestion
        self.filter_box = Filter(parent=self)
```

Better use keyword arguments, it eases future changes (like adding new parameters), and ease reading (no need to check the declaration to know what's passed).